src/features/Banner/index.js

```diff
@@ -0,0 +1,3 @@
+import component from './component';
+
+export default component;
```

Comment:
as discussed on the call, let's use real names of the components here and in the component file itself to keep names of them for Profiler